docs/source/eks.md

```diff
@@ -82,28 +82,30 @@ For this guide, we'll create an EKS cluster with the following:
       allow: true
 ```
 
-### Installing Required Tools
+### Prerequisites
 
 #### Installing script third party dependencies
 
 Script requires several dependencies:
-- Helm - See: https://docs.helm.sh/using_helm/#installing-helm
 - eksctl - See: https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html
 - kubectl - See: https://kubernetes.io/docs/tasks/tools/install-kubectl/
 
+#### Setting up local nodes with NodeConfig
 
-#### Install the local provisioner
+To run Scylla, it is most important, and necessary in production, that you convert the ephemeral storage's filesystem to `xfs`.
+`NodeConfig` performs the necessary RAID configuration and XFS filesystem creation, as well as it optimizes the nodes. You can read more about it in [Performance tuning](performance.md) section of ScyllaDB Operator's documentation.
 
-We deploy the local volume provisioner, which will discover their mount points and make them available as PersistentVolumes.
+Deploy `NodeConfig` to let it take care of the above operations:
 ```
-helm install local-provisioner examples/common/provisioner
+kubectl apply --server-side -f examples/gke/nodeconfig-alpha.yaml
 ```
 
-#### Deploy tuning DaemonSet
+#### Deploying Local Volume Provisioner
 
-Deploy tuning DaemonSet, this will configure your disks and apply several optimizations
+Afterwards, deploy ScyllaDB's [Local Volume Provisioner](https://github.com/scylladb/k8s-local-volume-provisioner), which will discover the RAID0 arrays' mount points and dynamically provision them as PersistentVolumes for your ScyllaDB cluster.
```

Comment:
> which will discover the RAID0 arrays' mount points

I don't think there is a discovery. you just need to hard code the same path into the config

Reply:
done